regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -122,9 +125,12 @@ var MainView = Backbone.View.extend({
         'search-results': SearchResultsView,
         'diff': DiffView,
         'appendix': RegView,
-        'interpretation': RegView
+        'interpretation': RegView,
+        'comment-review': CommentReviewView
     },
 
+    commentTypes: ['reg-section', 'preamble-section'],
```

Comment:
Agree, but `commentTypes` may be totally gone with the addition of write mode, so might suggest deferring this as well.